stacks/portstack.go

```diff
@@ -419,6 +431,7 @@ func (ps *PortStack) handleEth(dst []byte) (n int, err error) {
 	isDebug := ps.isLogEnabled(slog.LevelDebug)
 	socketPending := false
 	if ps.pendingUDPv4 > 0 {
+		
```

Comment:
Remove all whitepsace noise!

Reply:
I can't get format on save to work, but doing it manually with alt-shift-f does work - super annoying

Reply:
This is my VSCode JSON file. Find it by going to settings and clicking the page icon top right corner
```
{
    "workbench.colorTheme": "GitHub Dark High Contrast",
    "workbench.sideBar.location": "right",
    "[go]": {
        "editor.insertSpaces": false,
        "editor.formatOnSave": true,
        "editor.codeActionsOnSave": {
            "source.organizeImports": "explicit"
        }
    },
    "gopls": {
        "ui.semanticTokens": true
    },
    "go.toolsManagement.autoUpdate": true
}
```